core/src/plugins/access.swift/class.swiftAccessDriver.php

```diff
@@ -125,4 +155,10 @@ public function makeSharedRepositoryOptions($httpVars, $repository)
         return $newOptions;
     }
 
+    public function directoryUsage($directoryPath, $repositoryResolvedOptions)
```

Comment:
Bonjour Charles,

added that note,

Best regards,

Steve

---
Steve Ludovicy

Global Media Systems SA
212, route de Thionville -   L-2610 Howald
tel. +352 48 28 11   -   fax +352 48 28 110

On 2015-03-20 09:32, Charles du Jeu wrote:
> In core/src/plugins/access.swift/class.swiftAccessDriver.php [1]:
> 
>> @@ -125,4 +155,10 @@ public function
> makeSharedRepositoryOptions($httpVars, $repository)
>> return $newOptions;
>> }
>> 
>> + public function directoryUsage($directoryPath,
> $repositoryResolvedOptions)
> 
> Can you add a note that this implements only the top-level bucket
> bytesize, without taking into account the directoryPath?
> 
> --
> Reply to this email directly or view it on GitHub [2].
> 
> 	*
> 
> Links:
> ------
> [1] https://github.com/pydio/pydio-core/pull/874#discussion_r26824671
> [2] https://github.com/pydio/pydio-core/pull/874/files#r26824671